nc-news-fe/src/components/ArticleVoting.jsx

```diff
@@ -1,9 +1,25 @@
+import { useEffect, useState } from "react"
+
+import { patchVotesByArticleId } from "../../api"
+
 const ArticleVoting = ({currentArticle}) => {
+
+    const [votes, setVotes] = useState(0)
+
+    useEffect(()=>{
+        setVotes(currentArticle.votes)
+    }, [])

```

Comment:
Dont think this is this ticket, but you've made changes and included it in the commit so i'll comment. Im not sure why theres a useEffect to increment votes straight away. I presume it just increments it by zero so is abit pointless